file_parser.py

```diff
@@ -0,0 +1,135 @@
+from collections import Counter
+from typing import List, Tuple
+from enum import Enum
+import pandas as pd
+import morfeusz2
+import textract
+import spacy
+import nltk
+import sys
+import re
+import os
+
+
+# BEFORE YOU RUN THIS CODE:
+# nltk.download('wordnet') 
+# nltk.download('punkt') 
+# python -m spacy download pl_core_news_lg
+# python -m spacy download en_core_web_lg
+
+
+class Language(Enum):
+    ENGLISH = 1
+    POLISH  = 2
+
+def extract_text_from_file(file_path: str) -> str:
+    """ 
+    Returns plain text read from provided file.
+    Supported extensions: pdf, odt, docx, txt
+    """
+    if file_path.split(".")[-1] != "txt":
+        text = textract.process(file_path).decode('utf8')
+    else:
+        with open(file_path, "rb") as f:
+            text = f.read().decode('utf8')
+    return text
+
+def clear_text(text: str) -> str:
+    """ 
+    Clears the text from punctuation by replacing every character 
+    specified in the regrex with a space .
+    """
+    return re.sub('[^A-Za-z0-9ĘęÓóĄąŚśŁłŻżŹźĆćŃń ]+', ' ', text)
+
+def recognize_data_strings(text: str, language: Language) -> list:
+    """
+    Returns list of strings containing data recognized as important
+    such as dates, organization names, people's names and surnames and others.
+    """
+    # TODO We might want to return not only the phrases but the categories as well
+    important_phrases = []
+    if language == Language.ENGLISH:
+        model = "en_core_web_lg"
+    elif language == Language.POLISH:
+        model = "pl_core_news_lg"
+    nlp = spacy.load(model)
+    important_text = nlp(text)
+    for ent in important_text.ents:
+        important_phrases.append(ent.text)
+    return important_phrases
+
+def lemmatize(words: List[str], language: Language) -> List[str]:
```

Comment:
Using new syntax (from *Python 3.9* onward):
```suggestion
def lemmatize(words: list[str], language: Language) -> list[str]:
```